docker-compose.yml

```diff
@@ -90,9 +96,20 @@ services:
     links:
       - s3:s3.localhost
     logging: *default-logging
+    volumes: *default-volumes
     ports: []
     stdin_open: true
     tty: true
   redis:
     image: redis
```

Comment:
Same as above re: version numbers; also looks like there's an `-alpine` Redis image too.

Reply:
I saw a `postgres-alpine` as well when checking the tag versions, I'll definitely see if they work out of the box.